src/cpp/ext/filters/census/server_call_tracer.cc

```diff
@@ -169,17 +171,18 @@ class OpenCensusServerCallTracer : public grpc_core::ServerCallTracer {
     }
 
     switch (annotation.type()) {
-      case AnnotationType::kMetadataSizes:
```

Comment:
Are annotations used for anything other than tracing?  If not, this seems fine.  But if so, it seems like we shouldn't fail to add the annotation just because the RPC is not being sampled for tracing.

Reply:
I don't think annotations are in use today anywhere yet, so this seems to be a good time to change the default behavior to not produce strings, and let later annotations can decide what to do.

Reply:
yeah, we only use annotations for tracing. Seems fine to me